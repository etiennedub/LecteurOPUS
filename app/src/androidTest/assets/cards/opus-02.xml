<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<Opus>
  <ID>0000000000000004007100000000000092A4696202500A44001F2A41009000</ID>
  <Expiration>04789200091B94EA83A33260A0E2C1617F1500000000000000000000009000</Expiration>
  <Transit>
    <List index="0">721DF600000CF00028277802F8F7B90EFB0A20000000000000000000009000</List>
    <List index="1">5C426D80000CF00028032802F8F7AE2136CA20000000000000000000009000</List>
    <List index="2">5C352A80000CF0002809180560B7AE1A944A22000000000000000000009000</List>
  </Transit>
  <Subscription>
    <List index="0">1E0A092B80000007F48D6F0770000000000020000000000000000000009000</List>
    <List index="1">1E0A092B80000007F48D700A32000000000020000000000000000000009000</List>
    <List index="2">1E0A092B80000007F48D70E732000000000020000000000000000000009000</List>
  </Subscription>
  <Ticket>
    <List index="0">0000000000000000000000FFFFFF0000000000000000000000000000009000</List>
    <List index="1">0000000000000000000000FFFFFF0000000000000000000000000000009000</List>
    <List index="2">0000000000000000000000FFFFFF0000000000000000000000000000009000</List>
  </Ticket>
</Opus>
