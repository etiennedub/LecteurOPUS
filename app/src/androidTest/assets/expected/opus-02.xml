<?xml version='1.0' encoding='UTF-8' standalone='yes' ?><Card><Type>OPUS</Type><ID>2460248418</ID><ExpirationDate>31/01/2017</ExpirationDate><Trip><List index="0" Date="30/12/2016" Time="16:44" Transfert="false" OperatorId="20" BusId="95" OperatorName="RTC" BusName="METROBUS 800" /><List index="1" Date="02/03/2013" Time="20:43" Transfert="false" OperatorId="20" BusId="95" OperatorName="RTC" BusName="METROBUS 800" /><List index="2" Date="27/02/2013" Time="09:57" Transfert="false" OperatorId="20" BusId="172" OperatorName="RTC" BusName="METROBUS 803" /></Trip><Contract><List index="0" Subscription="false" Valid="false" NbTicket="0" OperatorId="20" /><List index="1" Subscription="false" Valid="false" NbTicket="0" OperatorId="20" /><List index="2" Subscription="false" Valid="false" NbTicket="0" OperatorId="20" /></Contract></Card>
